content/docs/guides/branching-intro.md

```diff
@@ -0,0 +1,97 @@
+
+<a href="/docs/guides/branching-neon-cli" description="Learn how to instantly create and manage branches with the Neon CLI" icon="cli">Branching with the Neon CLI</a>
+
+<a href="/docs/guides/branching-github-actions" description="Automate branching with Neon's GitHub Actions for branching" icon="split-branch">Branching with GitHub Actions</a>
+
+<a href="https://neon.tech/blog/automating-neon-branch-creation-with-githooks" description="Blog: Learn how to automating branch creation with Githooks" icon="split-branch">Branching with Githooks</a>
+
+</DetailIconCards>
+
+## Preview deployments
+
+Create a branch for each preview deployment with the Neon-Vercel Integration.
+
+<DetailIconCards>
+
+<a href="/docs/guides/branching-neon-api" description="Connect your Vercel project and create a branch for each preview deployment" icon="split-branch">The Neon Vercel Integration</a>
+
+<a href="https://neon.tech/blog/database-branching-for-postgres-with-neon" description="Blog: Read about full-stack preview deployments using the Neon Vercel Integration" icon="split-branch">Preview deployments with Vercel</a>
+
+<a href="https://neon.tech/blog/branching-with-preview-environments" description="Blog: A database for every preview environment with GitHub Actions and Vercel" icon="split-branch">A database for every preview</a>
+
+</DetailIconCards>
+
+## Test queries
+
+Test potentially destructive or performance-impacting queries before your run them in production.
+
+<DetailIconCards>
+
+<a href="/docs/guides/branching-test-queries" description="Instantly create a branch to test queries before running them in production" icon="queries">Branching — Testing queries</a>
+
+</DetailIconCards>
+
+## Data recovery
+
+Create a branch from the past to recover lost data or track down issues.
+
+<DetailIconCards>
+
+<a href="/docs/introduction/point-in-time-restore" description="Learn about Neon's point-in-time restore feature" icon="invert">Point-in-time restore</a>
+
+<a href="/docs/guides/branching-pitr" description="Learn how to restore your data to previous state using Neon's branching feature" icon="screen">Create a branch from the past</a>
+
+<a href="https://neon.tech/blog/time-travel-with-postgres" description="Blog: Time travel with database branching to recover lost data" icon="screen">Time Travel with Postgres</a>
+
+</DetailIconCards>
+
+## Branching guides
+
+Learn how to refresh a branch or promote a branch to become your primary branch.
+
+<DetailIconCards>
+
+<a href="/docs/guides/branch-refresh" description="Learn how to refresh a branch with the latest data using the Neon API" icon="split-branch">Refresh a branch</a>
+
+<a href="/docs/guides/branch-promote" description="Promote a branch to the primary branch of your Neon project using the Neon API" icon="split-branch">Promote a branch to primary</a>
```

Comment:
Something about "promote a branch to the primary branch" gives me pause, since "the" is so definitive, but the action is more about changing statuses. Consider something like (though this doesn't sound perfect either):

"Promote a branch to primary status..."

Reply:
I went with shortening it to "Promote a branch", otherwise the title is too long and makes a new line in the card.